docs/reference/connectors/bench.md

```diff
@@ -1,194 +0,0 @@
----
-sidebar_label: bench (Benchmarking)
```

Comment:
this is what is missing.

Reply:
the problem with introducing those headers is that we're now having docusaurus specific syntax in the comments that'd probably not play nice with rust doc generation, is having the direct names really a big issue?